builder/nrfutil.go

```diff
@@ -0,0 +1,26 @@
+package builder
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/tinygo-org/tinygo/compileopts"
+)
+
+// https://infocenter.nordicsemi.com/index.jsp?topic=%2Fug_nrfutil%2FUG%2Fnrfutil%2Fnrfutil_intro.html
+
+func makeDFUFirmareImage(options *compileopts.Options, infile, outfile string) error {
+	cmdLine := []string{"nrfutil", "pkg", "generate", "--hw-version", "52", "--sd-req", "0x0", "--debug-mode", "--application", infile, outfile}
+
+	if options.PrintCommands {
+		fmt.Printf("%s\n ", strings.Join(cmdLine, " "))
+	}
+
+	cmd := exec.Command(cmdLine[0], cmdLine[1:]...)
+	err := cmd.Run()
+	if err != nil {
+		return fmt.Errorf("could not run nrfutil pkg generate: %w", err)
+	}
+	return nil
+}
```

Comment:
The following error message was output.

```
C:\tinygo\tinygo>tinygo flash --target pca10059 --size short -x -port COM33 ./src/examples/blinky1
   code    data     bss |   flash     ram
   7368      32    4080 |    7400    4112
nrfutil pkg generate --hw-version 52 --sd-req 0x0 --debug-mode --application C:\Users\sago3\AppData\Local\Temp\tinygo022542263\main.hex C
:\Users\sago3\AppData\Local\Temp\tinygo022542263\main.zip
 error: could not run nrfutil pkg generate: exit status 4294967295
```

It seems that nrfutil on windows needs to handle standard output.
I downloaded nrfutil from the following URL

https://github.com/NordicSemiconductor/pc-nrfutil/releases/tag/v6.1

```
$ nrfutil.exe version
nrfutil version 6.1.0
```

So, I don't know the details, but I would like to add the following to `tinygo flash` from windows.

```suggestion
import (
	"fmt"
	"io/ioutil"
	"os/exec"
	"strings"

	"github.com/tinygo-org/tinygo/compileopts"
)

// https://infocenter.nordicsemi.com/index.jsp?topic=%2Fug_nrfutil%2FUG%2Fnrfutil%2Fnrfutil_intro.html

func makeDFUFirmareImage(options *compileopts.Options, infile, outfile string) error {
	cmdLine := []string{"nrfutil", "pkg", "generate", "--hw-version", "52", "--sd-req", "0x0", "--debug-mode", "--application", infile, outfile}

	if options.PrintCommands {
		fmt.Printf("%s\n ", strings.Join(cmdLine, " "))
	}

	cmd := exec.Command(cmdLine[0], cmdLine[1:]...)
	// It seems that nrfutil on windows needs to handle standard output
	cmd.Stdout = ioutil.Discard
	err := cmd.Run()
	if err != nil {
		return fmt.Errorf("could not run nrfutil pkg generate: %w", err)
	}
	return nil
}
```

Reply:
Great thanks! Not sure why my windows installation wasn't picky about that